magellan/tests/test_attr_equiv_blocker.py

```diff
@@ -0,0 +1,230 @@
+import os
+from nose.tools import *
+import pandas as pd
+import unittest
+
+import magellan as mg
+
+p = mg.get_install_path()
+path_for_A = os.sep.join([p, 'datasets', 'table_A.csv'])
+path_for_B = os.sep.join([p, 'datasets', 'table_B.csv'])
+l_key = 'ID'
+r_key = 'ID'
+l_block_attr_1 = 'zipcode'
+l_block_attr_2 = 'birth_year'
+l_block_attr_3 = 'name'
+r_block_attr_1 = 'zipcode'
+r_block_attr_2 = 'birth_year'
+r_block_attr_3 = 'name'
+l_output_attrs = ['zipcode', 'birth_year']
+r_output_attrs = ['zipcode', 'birth_year']
+l_output_prefix = 'l_'
+r_output_prefix = 'r_'
+
+bogus_attr = 'bogus'
+block_attr_multi = ['zipcode', 'birth_year']
+
+
```

Comment:
@sanjibkd, the code looks fine to me. Can you pl. add the unit test cases mentioned, commit and push to the branch. 

You dont have to create a pull request. The current pull request will get automatically updated if you push to scaleup_AE_blocker branch.